lib/web/apiserver_test.go

```diff
@@ -8538,9 +8538,9 @@ func createProxy(ctx context.Context, t *testing.T, proxyID string, node *regula
 		},
 	)
 	handler.handler.cfg.ProxyKubeAddr = utils.FromAddr(kubeProxyAddr)
+	handler.handler.cfg.PublicProxyAddr = webServer.Listener.Addr().String()
```

Comment:
About this change: this was a nasty bug in the test web pack.
The public proxy address is supposed to be "hostname:port" but the test pack was putting a URL in there. The value was correctly set in real-world builds.

In teleport's codebase, Addr should always mean `hostname:port`.